File: software/examples/perl/example_simple_voltage.pl
#!/usr/bin/perl

use strict;
use Tinkerforge::IPConnection;
use Tinkerforge::BrickletIndustrialAnalogOutV2;

use constant HOST => 'localhost';
use constant PORT => 4223;
use constant UID => 'XYZ'; # Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0

my $ipcon = Tinkerforge::IPConnection->new(); # Create IP connection
my $iao = Tinkerforge::BrickletIndustrialAnalogOutV2->new(&UID, $ipcon); # Create device object

$ipcon->connect(&HOST, &PORT); # Connect to brickd
# Don't use device before ipcon is connected

# Set output voltage to 3.3V
$iao->set_voltage(3300);
$iao->set_enabled(1);

print "Press key to exit\n";
<STDIN>;
$iao->set_enabled(0);
$ipcon->disconnect();

File: software/examples/ruby/example_simple_voltage.rb
#!/usr/bin/env ruby
# -*- ruby encoding: utf-8 -*-

require 'tinkerforge/ip_connection'
require 'tinkerforge/bricklet_industrial_analog_out_v2'

include Tinkerforge

HOST = 'localhost'
PORT = 4223
UID = 'XYZ' # Change XYZ to the UID of your Industrial Analog Out Bricklet 2.0

ipcon = IPConnection.new # Create IP connection
iao = BrickletIndustrialAnalogOutV2.new UID, ipcon # Create device object

ipcon.connect HOST, PORT # Connect to brickd
# Don't use device before ipcon is connected

# Set output voltage to 3.3V
iao.set_voltage 3300
iao.set_enabled true

puts 'Press key to exit'
$stdin.gets

iao.set_enabled false

ipcon.disconnect
